sujeong/app.js

```diff
@@ -5,7 +5,7 @@ require("dotenv").config();
 const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
-const { DataSource } = require("typeorm");
+const { DataSource, AbstractRepository, And } = require("typeorm");
```

Comment:
AbstractRepository, And 해당 모듈을 사용하지 않을 것 같은데, 제거해 주세요.

```suggestion
const { DataSource } = require("typeorm");
```